server/controllers/officeControllers.js

```diff
@@ -4,7 +4,7 @@ import officesDb from '../models/officeModels';
  * Class representing OfficeController
  * @class OfficeController
  */
-class OfficesController {
```

Comment:
Your controller should be in singular, not plural; *OfficeController*

Reply:
Acknowledged with thanks.
I have implemented the changes based on the project naming convention.